src/hooks/useDragAndDropSubThought.ts

```diff
@@ -162,19 +165,26 @@ const dropCollect = (monitor: DropTargetMonitor) => ({
   // is being hovered over current thought irrespective of whether the given item is droppable
   isBeingHoveredOver: monitor.isOver({ shallow: true }),
   isDeepHovering: monitor.isOver(),
+  canDropThought: monitor.canDrop(),
 })
 
 /** A draggable and droppable SubThought hook. */
 const useDragAndDropSubThought = (props: Partial<DroppableSubthoughts>) => {
   const propsTypes = props as DroppableSubthoughts
 
-  const [{ isHovering, isBeingHoveredOver, isDeepHovering }, dropTarget] = useDrop({
+  const [{ isHovering, isBeingHoveredOver, isDeepHovering, canDropThought }, dropTarget] = useDrop({
     accept: [DragAndDropType.Thought, NativeTypes.FILE],
     canDrop: (item, monitor) => canDrop(propsTypes, monitor),
     drop: (item, monitor) => drop(propsTypes, monitor),
     collect: dropCollect,
   })
 
+  // Encapsulate the useHoveringPath and useDragLeave hooks
+  if (props.path) {
```

Comment:
The `Partial<DroppableSubthoughts>` definition was an odd way to define the parameters, so I fixed it in 3e1f3db06fc. That obviates the need for this condition.

FYI Hooks are not allowed in conditions. 
See: https://legacy.reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level